app/controllers/food_controller.rb

```diff
@@ -0,0 +1,5 @@
+class FoodController < ApplicationController
```

Comment:
In order for routing to work, you'll have to make this FoodsController with an s because your resource in routes is titled `foods`